src/atlantis/manager/helper/helper.go

```diff
@@ -192,7 +193,15 @@ func RegionAndZone(zone string) string {
 }
 
 func ZoneMinusRegion(zone string) string {
-	return strings.Replace(zone, Region, "", 1)
+	zmr := strings.Replace(zone, Region, "", 1)
+	if zmr == "" {
+		results := strings.Split(zone, "-")
+		if len(results) == 2 {
+			zmr = results[1]
+		}
+	}
+	log.Printf("ZoneMinusRegion: zone=%s Region=%s zmr='%s'", zone, Region, zmr)
```

Comment:
It's only called when we need to register a router so I don't think you'll see the log that often.